tools/SdkVersionVerifier/VersionComparer.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace SdkVersionVerifier;
+
+internal static class VersionComparer
+{
+    public static bool CompareVersions(DotnetSdkVersion expectedDotnetSdkVersion, DotnetSdkVersion version)
+    {
+        return CompareVersions(expectedDotnetSdkVersion, version.Net6SdkVersion, version.Net7SdkVersion, version.Net8SdkVersion);
+    }
+
+    public static bool CompareVersions(
+        DotnetSdkVersion expectedDotnetSdkVersion,
+        string? net6SdkVersion,
```

Comment:
could be more dynamic, so new sdk versions will not cause issues?

Reply:
I considered other approaches, but decided to start simple. I'd prefer to include improvements in a separate PR, unless they are urgent.